composer.json

```diff
@@ -871,7 +871,8 @@
     "swiftmailer/swiftmailer": "^5.4.5"
   },
   "require-dev": {
-    "codeception/codeception": "^4.0.0",
+    "codeception/codeception": "*",
+    "codeception/lib-innerbrowser": "dev-feature/use-client-through-getter as 1.3.2",
```

Comment:
I dont think we can do that, we use
    {
      "type": "git",
      "url": "git@github.com:spryker-sdk/lib-innerbrowser.git"
    },
on nonsplit as repo hack

We then need to also make our own release here now.

Reply:
Shouldnt this be resolved, as well?

Reply:
Done in separate PR